### DISCUSSION TITLE
Should `s_corp_self_employment_income` be included in `partnership_s_corp_income` and / or gross income?

### DISCUSSION THREAD

**Discussion by PavelMakarchuk** [PolicyEngine/policyengine-us#5271] [General]
Currently `s_corp_self_employment_income` is only used in `taxable_self_employment_income` while `partnership_s_corp_income` is used for the QBID income definitions - would we need to consolidate those which will also might require adjustments in US-Data

**Comment by MaxGhenis** [comment]
After further research I've filed #5272 and #5273 on this. It will also require changes to policyengine-us-data, and those issues link to relevant issues there.
